### PR TITLE
Offer screen continued

### PR DESCRIPTION
**Pull request by gustafguner**
- Perils section for mobile
- Peril modal content for desktop + mobile
- Compare table section for mobile

Screenshot 2019-11-26 at 15 52 43
Screenshot 2019-11-26 at 16 16 27
Screenshot 2019-11-26 at 16 16 49

### CODE REVIEW

File: src/pages/OfferNew/Perils/PerilCollection/index.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  setIsShowingPeril: (isShowingPeril: boolean) => void
+}
+
+const Wrapper = styled('div')`
+  width: 100%;
+  display: flex;
+  flex-flow: row wrap;
+`
+
+export const PerilCollection: React.FC<Props> = ({
```
Comment (hedvigoscar) [inline src/pages/OfferNew/Perils/PerilCollection/index.tsx]: `React.FC<T>` 🤯

Stealing this for sure

File: src/pages/OfferNew/Perils/PerilSwiper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+const PERILS_PER_SLIDE = 4
+const PERILS_PER_ROW = PERILS_PER_SLIDE / 2
+
+interface Props {
+  perils: Peril[]
+  setCurrentPeril: (index: number) => void
+  setIsShowingPeril: (isShowingPeril: boolean) => void
+}
+
+const Wrapper = styled.div``
```
Comment (hedvigoscar) [inline src/pages/OfferNew/Perils/PerilSwiper/index.tsx]:
```suggestion
const Wrapper = styled.div``
```

🤔

Comment (gustafguner) [reply src/pages/OfferNew/Perils/PerilSwiper/index.tsx]: Whoops, left-overs 😎

File: src/pages/OfferNew/Perils/PerilCollection/index.tsx (hunk 1 above)
Comment (gustafguner) [reply src/pages/OfferNew/Perils/PerilCollection/index.tsx]: Yeah, it's very neat

**Comment by gustafguner** [comment]
> Looks fantastic! Can you add a few screenshots for reference? 🙂

Sure, coming up! 📸

**Pull request merged**